Refine Country Setting by API rules (#6070)
braze-inc/braze-docs: _docs/_api/objects_filters/user_attributes_object.md

| alias_name | (string) |
| alias_label | (string) |
| braze_id | (string, optional) When a user profile is recognized via the SDK, an anonymous user profile is created with an associated `braze_id`. The `braze_id` is automatically assigned by Braze, cannot be edited, and is device-specific. | 

> | country | (string) We require that country codes be passed to Braze in the [ISO-3166-1 alpha-2 standard][17]. Our API will make a best effort to map countries recieved in different formats. "Australia" to "AU" for example. If we are unable to match the input to a given [ISO-3166-1 alpha-2 standard][17] the country value will be set to NULL |

Reviewer: ```suggestion
| country | (string) We require that country codes be passed to Braze in the [ISO-3166-1 alpha-2 standard][17]. Our API will make a best effort to map countries received in different formats. For example, "Australia" may map to "AU." However, if the input doesn't match a given [ISO-3166-1 alpha-2 standard][17], the country value will be set to `NULL`. |
```

> | country | (string) We require that country codes be passed to Braze in the [ISO-3166-1 alpha-2 standard][17]. Our API will make a best effort to map countries received in different formats. For example, "Australia" may map to "AU." However, if the input doesn't match a given [ISO-3166-1 alpha-2 standard][17], the country value will be set to `NULL`. |

Reviewer: ```suggestion
| country | (string) We require that country codes be passed to Braze in the [ISO-3166-1 alpha-2 standard][17]. Our API will make a best effort to map countries received in different formats. For example, "Australia" may map to "AU". However, if the input doesn't match a given [ISO-3166-1 alpha-2 standard][17], the country value will be set to `NULL`. |
```
